modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/ImportObjectDefinitionMVCActionCommand.java

```diff
@@ -56,52 +57,52 @@ protected void doProcessAction(
 			ActionRequest actionRequest, ActionResponse actionResponse)
 		throws Exception {
 
-		try {
-			_importObjectDefinition(actionRequest);
-		}
-		catch (Exception exception) {
-			if (_log.isDebugEnabled()) {
-				_log.debug(exception);
-			}
+		JSONObject jsonObject = _importObjectDefinition(actionRequest);
```

Comment:
@G-Sa do we really need to maintain this method that is used only once?

Reply:
This method was always just being used once. We could extract it, but i think it would make the code way less readable, especially now that we introduced the ability to handle lists